Makefile

```diff
@@ -0,0 +1,25 @@
+FLIT ?= flit
+
+SPHINX_BUILD ?= sphinx-build
+SPHINX_BUILD_SOURCE_DIR ?= doc
+SPHINX_BUILD_BUILD_DIR ?= build/sphinx
+
+.PHONY: man
+man:
+	$(SPHINX_BUILD) -a -b man $(SPHINX_BUILD_SOURCE_DIR) $(SPHINX_BUILD_BUILD_DIR)
+
+.PHONY: html
+html:
+	$(SPHINX_BUILD) -a -b html $(SPHINX_BUILD_SOURCE_DIR) $(SPHINX_BUILD_BUILD_DIR)
+
+.PHONY: sdist
+sdist:
+	$(FLIT) build --format sdist
+
+.PHONY: wheel
+wheel:
+	$(FLIT) build --format wheel
```

Comment:
I'd use `build` rather than `flit` here, if only to exercise what's closer to the more common PEP517 workflow.